sdk/eventhub/event-hubs/test/hubruntime.spec.ts

```diff
@@ -162,11 +238,51 @@ describe("RuntimeInformation #RunnableInBrowser", function(): void {
       try {
         const controller = new AbortController();
         setTimeout(() => controller.abort(), 1);
-        await client.getPartitionProperties("0", controller.signal);
+        await client.getPartitionProperties("0", {
+          abortSignal: controller.signal
+        });
         throw new Error(`Test failure`);
       } catch (err) {
         err.message.should.match(/The [\w]+ operation has been cancelled by the user.$/gi);
       }
     });
+
+    it("can be manually traced", async function(): Promise<void> {
+      const tracer = new TestTracer();
+      setTracer(tracer);
+
+      const rootSpan = tracer.startSpan("root");
+      client = new EventHubClient(service.connectionString, service.path);
+      const partitionRuntimeInfo = await client.getPartitionProperties("0", {
+        parentSpan: rootSpan
+      });
+      partitionRuntimeInfo.partitionId.should.equal("0");
+      partitionRuntimeInfo.eventHubName.should.equal(service.path);
+      partitionRuntimeInfo.lastEnqueuedTimeUtc.should.be.instanceof(Date);
+      should.exist(partitionRuntimeInfo.lastEnqueuedSequenceNumber);
+      should.exist(partitionRuntimeInfo.lastEnqueuedOffset);
+      rootSpan.end();
+
+      const rootSpans = tracer.getRootSpans();
+      rootSpans.length.should.equal(1, "Should only have one root span.");
+      rootSpans[0].should.equal(rootSpan, "The root span should match what was passed in.");
+
+      const expectedGraph: SpanGraph = {
+        roots: [
+          {
+            name: rootSpan.name,
+            children: [
+              {
+                name: "Azure.EventHubs.getPartitionProperties",
+                children: []
+              }
+            ]
+          }
+        ]
+      };
+
+      tracer.getSpanGraph(rootSpan.context().traceId).should.eql(expectedGraph);
+      tracer.getActiveSpans().length.should.equal(0, "All spans should have had end called.");
+    });
   });
 }).timeout(60000);
```

Comment:
kinda curious why the timeout is so big. I feel like we have a few tests that need to be sped up through mocking (though don't worry about this for this PR)

Reply:
Yes, this is because currently all the tests are live tests. I'll be taking a look at mocking in the near future :)